src/App.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import './App.css';
 
+import GoogleLogin from 'react-google-login';
+
+import Login from './Component/Login';
+import Display from './Component/Display';
```

Comment:
Are these components? Can we categorise them better?

Reply:
Yes Like Homepage and Dashboard?